HackerRank/Algorithms/implementation/The_Time_in_Words.java

```diff
@@ -1,6 +1,4 @@
 package implementation;
-
-
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStreamReader;
```

Comment:
Since you used scanner these 3 imports can be removed right ? Also don't we need to import `java.util.Scanner` ?

Reply:
Also can indent line 16 correctly ?

Reply:
Everything should be fixed. Sorry I removed java.util.Scanner, I accidentally deleted it.